.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "10"
+  - "12"
+cache:
+  yarn: true
+script:
+  - yarn test
+  
```

Comment:
This line should be empty and contain no space 😇

Reply:
Done, thanks 👍